lib/touchbar.es

```diff
@@ -92,13 +92,8 @@ export const touchBar = new TouchBar({
   escapeItem: poibutton,
 })
 //Change Volume btn
-export const touchBarreinit = (muted) => {
-  if (muted) {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-off.png')
-  }
-  else {
-    volume.icon = path.join(ROOT, 'assets', 'img', 'touchbar', 'volume-up.png')
-  }
+export const touchBarreinit = () => {
```

Comment:
Could this be named touchbarReinit or touchBarReinit?

btw, as you may see, many methods are in the form verb + noun, e.g. readFile. Personally I'm in favor of that